tests/contact-controller.test.js

```diff
@@ -6,23 +6,29 @@ const endOfDay = require('date-fns/endOfDay');
 const createContactController = require('../controllers/contact-controller');
 const ContactBuilder = require('./contact-builder');
 
+const callTypes = {
+  child: 'Child calling about self',
+  caller: 'Someone calling about a child',
+};
+
 const { Op } = Sequelize;
 const DBConnectionMock = new SequelizeMock();
 const MockContact = DBConnectionMock.define('Contacts');
 
+const ContactController = createContactController(DBConnectionMock);
```

Comment:
Was there a need to define this in each test before?  Is there any risk that reusing the mock can have unexpected behavior?  I'm just wondering if there was a reason why @murilovmachado defined it separate before.  I don't understand the mocks here well enough to know for myself.

Reply:
I've just tested it and it behaves just fine reusing the `ContactController`. Implementing these tests and mocks, at first, was a bit of trial and error. That's probably the reason why we had `ContactController` defined in each individual method. But reusing it as @GPaoloni did seems to be a better option.